README.md

```diff
@@ -0,0 +1,38 @@
+# Ibanity signature service
+
+This service can be ran locally, in a Kubernetes cluster, ... to easily create your signatures to call the different APIs offered by Ibanity. See https://documentation.ibanity.com/security#http-signature for more info on the siganture format.
+
+## Setting up the service
+
+Enclosed in this project is a .env.template. Copy this file and name it .env, then fill in the variables with your correct values (not the ones in attached screenshot).
+
+![alt text](./images/.env-example.png)
```

Comment:
`alt text` is a placeholder for what you want to display if the image is broken for some reason. Maybe
```suggestion
![env example image](./images/.env-example.png)
```